Cork/Views/Start Page/Start Page.swift

```diff
@@ -93,26 +93,26 @@ struct StartPage: View
                                 {
                                     GroupBoxHeadlineGroup(
                                         image: "terminal",
-                                        title: "You have \(brewData.installedFormulae.count) Formulae installed",
-                                        mainText: "Formulae are usually apps that you run in a terminal"
+                                        title: "\(brewData.installedFormulae.count) Formulae installed",
+                                        mainText: "Formulae are packages containing command-line tools"
```

Comment:
There's slight inconsistency where we say "Taps are sources of packages" & "Install package", but then the explainer/description for the Formulae & Casks sections never references a 'package'

Reply:
'Command-line tools' also seems to be Apple's preferred wording

Reply:
I don't think we should remove the "You have" part, starting a sentence with a number is a big no-no in writing.

`Formulae are packages containing command-line tools` is not good wording in my opinion, because even graphical apps can contain command-line tools. If we want to include the word "package", what about "Formulae are packages that you use through the Terminal"

Similar to above, I don't like how `Casks are packages containing graphical (GUI) apps` contains parentheses and an abbreviation that non-technical users don't know. Maybe "Casks are generally packages that have graphical windows"?

I don't like the change to `Taps are sources of packages that are not provided by Homebrew`, because when you're using Cork, you're interacting with Homebrew, so technically, Homebrew is providing even the packages that are not included in the "base install". Maybe "Taps provide additional packages"?

Reply:
> starting a sentence with a number is a big no-no in writing

I feel like I've seen this kind of short sentence leading with a number in various places though – maybe not the best examples but iOS storage settings "64GB of 128GB used" and Xcode settings Platforms tab "12 GB on disk" come to mind. Happy to reinstate "You have" nonetheless, I'm not strongly attached.

> "Formulae are packages that you use through the Terminal"

Will change!

> contains parentheses and an abbreviation that non-technical users don't know

In hindsight, this makes total sense. I don't think my wording was elegant at all here 😅 

> Maybe "Casks are generally packages that have graphical windows"?

Will change this too

> "Taps provide additional packages"

Short and sweet, sounds better!

Reply:
> I feel like I've seen this kind of short sentence leading with a number in various places though – maybe not the best examples but iOS storage settings "64GB of 128GB used" and Xcode settings Platforms tab "12 GB on disk" come to mind.

You're right, I think the main difference that in these examples, it's just a piece of data that's a part of a larger section (so that's why it's okay), while in our usage, we use these sentences as the section header and a kind of an explanation, so we need to follow sentence rules. Just a little tidbit, when I was working as a translator at Apple, I got instructed that the part "64GB of 128GB used" that you see in the iPhone Settings app is actually not considered correct anymore, and they're only keeping it because users are already familiar with it 😅 

> Will change! Perhaps 'a terminal' though given other terminal apps exist (iTerm etc)

I think that's a great idea

> In hindsight, this makes total sense. I don't think my wording was elegant at all here 😅

It's okay, I'm always here to help 🤪

> Short and sweet, sounds better!

Amazing! Let's do it.

Reply:
Ooh, interesting tidbit!